### DISCUSSION TITLE
Which UI is better to tell user their device has no internet connection?

### DISCUSSION THREAD

**Discussion by suesitran** [suesitran/interactive_diary#31] [Polls]
In case of Internet connection is OFF, which way is better to notify user that they're using the app without internet connection?

### Option 1:
<img src="https://user-images.githubusercontent.com/17781268/186081201-95d6ae59-73e6-4729-b364-8e430fdc9a45.png" width=400/>

### Option 2:
<img src="https://user-images.githubusercontent.com/17781268/186080484-30135f03-99e1-4e95-b158-b25254bf6888.png" width=400/>

**Comment by DongNguyenQ** [comment]
I prefers option 2. 

- About User:
   - it's more straightforward. User can get what is happening without processing some information from visual on option 1. 
   - On option 1. User have to guess what that icon stand for, and then ask question, does the X icon mean that i have no internet connection ? 

- About Design:
  - I think we don't need an icon for internet connection status here. Because it's rarely happens (this is just my assumption base on where i lives - VietNam). I don't know that it likes in another country. 
  - I don't know what informations this icon provide to user. But if it just show on/off status, then it doesn't provide so much informations for user.
